frontend/src/components/NavBar/index.tsx

```diff
@@ -48,19 +51,31 @@ function NavBar({ classes }: NavBarProps) {
     </Grid>
   ));
 
+  const title = get(appConfig, 'title', 'PRISM');
+  const subtitle = get(appConfig, 'subtitle', null);
+  const logo = get(appConfig, 'logo', null);
+
   return (
     <AppBar position="static" className={classes.appBar}>
       <Toolbar variant="dense">
         <Grid container>
           <Grid item xs={3} className={classes.logoContainer}>
-            <Typography
-              variant="h6"
-              className={classes.logo}
-              component={Link}
-              to="/"
-            >
-              {t('Prism')}
-            </Typography>
+            {logo && <img className={classes.logo} src={logo} alt="logo" />}
+            <Box display="flex" flexDirection="column">
+              <Typography
+                variant="h6"
+                className={classes.title}
+                component={Link}
+                to="/"
+              >
+                {t(title)}
```

Comment:
@HarryMytilinaios the English version can be passed as the key directly as it is done here